### ISSUE TITLE
What's up with the Hackage build matrix!?

### ISSUE THREAD

**Issue by andreasabel** [skogsbaer/HTF#70]
According to https://matrix.hackage.haskell.org/#/package/HTF, the HTF fails to build on every single ghc version!?
(Got here because of #69).

**Comment by skogsbaer** [comment]
I will have a look...

**Comment by skogsbaer** [comment]
I pushed some fixes relaxing upper bounds (see #73) but I doubt that this will fix the hackage build matrix. I usually build and test HTF with stack, also travis CI runs with stack. Currently, the CI checks run against GHC 7.10, 8.2, 8.4 and, since #73, also against GHC 8.6. The hackage build matrix reports the following error:

```Configuring library for HTF-0.13.2.4..
cabal: The following package dependencies were requested
    --dependency='cpphs=cpphs-1.20.8-1153542665a1da4e80687fc101b697ec404dd17b05e3c97e7ce0baa89a2f5dde'
however the given installed package instance does not exist.

cabal: Failed to build HTF-0.13.2.4 (which is required by exe:htfpp from HTF-0.13.2.4). The failure occurred during the configure step.```

Somehow, I don't think that this is a problem with HTF. I'm closing this issue now. #64 has already been fixed, I will release a version on hackage shortly.
